joomla/joomla-cms
# add redirect option after article submission

### Summary of Changes
If you create a frontend article submission, you cannot specify, where the user will be redirected after the submission. As default, the user will be redirected to the frontpage. With this patch, you can define a menu item where the user should be redirected or still use the default behavior.

### Testing Instructions
=> Create a frontend submission menu item
=> Look in the tab "options"
=> There should be no parameter
=> Submit an article in the frontend, you'll redirected to the front page
=> Apply patch
=> Look into the menu item options again, there should be a new parameter
=> Let it at "default" and submit a new article, the redirect to the frontpage should still work
=> Choose a new redirect target in the menu item option
=> Submit an article
=> The user should be redirected to the new (defined) target
